package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@sonata-project/admin-bundle",
+  "version": "0.1.0",
```

Comment:
Why 0.1.0? Why not the current version of sonata-admin package?

Reply:
We're little stuck on the asset problem. We can't use the current composer project version for the assets, because there we have no specific release process for the assets and they're not changing very often.